recipes/maxbin2/meta.yaml

```diff
@@ -0,0 +1,28 @@
+build:
+  number: 0
+
+package:
+  name: maxbin2
+  version: "2.2.1"
+source:
+  fn: MaxBin-2.2.1.tar.gz
+  url: http://downloads.jbei.org/data/microbial_communities/MaxBin/getfile.php?MaxBin-2.2.1.tar.gz
+  md5: 3748e8f65b5bf18f8bd341210724cc5c
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm # [osx]
+  run:
+    - libgcc # [linux]
+    - perl
+    - bowtie2
+    - fraggenescan >=1.30
+    - hmmer
+    - idba
+test:
+  commands:
+    - MaxBin > /dev/null
```

Comment:
I had this problem with #2618 and we solved it by testing if the return code was 255.
Providing reads as input would make a functional test but this is not the bioconda's aim (cf. also #2618)